cmd/podman/containers/unmount.go

```diff
@@ -20,11 +20,12 @@ var (
   An unmount can be forced with the --force flag.
 `
 	unmountCommand = &cobra.Command{
-		Use:     "unmount [options] CONTAINER [CONTAINER...]",
-		Aliases: []string{"umount"},
-		Short:   "Unmounts working container's root filesystem",
-		Long:    description,
-		RunE:    unmount,
+		Annotations: map[string]string{registry.EngineMode: registry.ABIMode},
```

Comment:
Is this saying unmount is supported in podman-remote?

Reply:
No, only "local"

Reply:
What I find confusing is that Annotations indicates which engines to use, now rather then Mode, is that basically the change that is going on?